doc/rst/source/grdmath.rst

```diff
@@ -128,6 +129,9 @@ Optional Arguments
 .. |Add_-di| unicode:: 0x20 .. just an invisible code
 .. include:: explain_-di.rst_
 
+.. |Add_-e| unicode:: 0x20 .. just an invisible code
+.. include:: explain_-e.rst_
+
 .. |Add_-f| unicode:: 0x20 .. just an invisible code
 .. include:: explain_-f.rst_
```

Comment:
-g may be needed here as well.

Reply:
Yes, added -g to the docs for consistency.